core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/EntityScopeProviderImpl.java

```diff
@@ -115,7 +115,6 @@ public Scope getScopeForNode(final Integer nodeId) {
             if (node.getAssetRecord() != null) {
                 Scope assetScope = new ObjectScope<>(Scope.ScopeName.NODE, node.getAssetRecord())
                         .map(ASSET, "category", (a) -> Optional.ofNullable(a.getCategory()))
-                        .map(ASSET, "vendor", (a) -> Optional.ofNullable(a.getVendor()))
```

Comment:
Why was this removed?

Reply:
Because two lines later again the vendor is set.